pymllm/bench_serve.py

```
@@ -0,0 +1,395 @@
+    print("pymllm-server benchmark (llama-bench style)")
+    print(f"  URL            : {args.url}")
+    print(f"  Model          : {args.model}")
+    print(f"  Prefill sizes  : {sizes}")
+    print(f"  Decode tokens  : {args.decode_tokens}")
+    print(f"  Repeat / warmup: {args.repeat} / {args.warmup}")
+    print(f"  Temperature    : {args.temperature}")
+
+    try:
+        r = requests.get(f"{args.url}/health", timeout=5)
+        print(f"  Server health  : {r.status_code}")
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)